source/extensions/common/wasm/wasm.cc

```diff
@@ -1,5 +1,3 @@
-#include "extensions/common/wasm/wasm.h"
```

Comment:
Per convention, header for the given file is always included at the top (see `./tools/check_format.py fix`).

Reply:
FYI silentdai, run "./tools/check_format_py check" to see if your PR is following the style rules.

Reply:
Hmm, this PR is modifying file is wasm.h, not "wavm.cc". And the include file is removed due to forward declaration.

And thank you for the reference of check_format. I did run it. See my initial comment.

Sorry for the unclear

Reply:
Oops. My above comment in this thread is wrong. Let me fix it.

Reply:
@PiotrSikora I figure out how I made this change... This change is introduced by 
```
./tools/check_format.py fix source/extensions/common/wasm/
```
See below
`
lambdai@lambdai-glaptop:~/workspace/envoy$ head -1 source/extensions/common/wasm/wasm.cc
#include "extensions/common/wasm/wasm.h"

lambdai@lambdai-glaptop:~/workspace/envoy$ ./tools/check_format.py check source/extensions/common/wasm/
ERROR: From source/extensions/common/wasm//wasm.cc
ERROR: header_order.py check failed for file: source/extensions/common/wasm//wasm.cc
ERROR:   source/extensions/common/wasm//wasm.cc:1
ERROR:   source/extensions/common/wasm//wasm.cc:24
ERROR: check format failed. run 'tools/check_format.py fix'

lambdai@lambdai-glaptop:~/workspace/envoy$ ./tools/check_format.py fix source/extensions/common/wasm/

lambdai@lambdai-glaptop:~/workspace/envoy$ head -1 source/extensions/common/wasm/wasm.cc 
#include <stdio.h>
`
@PiotrSikora  Let's talk offline

Reply:
`./tools/check_format.py fix` (on the full repository doesn't complain. 
The latest commit reverts this change.